connection.js

```diff
@@ -305,19 +335,24 @@ class Connection {
             return { reject: true, msg: "The " + notification.rules + " rules are not allowed for this bot, please choose allowed rules such as chinese rules. " };
         }
 
-        // ** first we eliminate absolute and simple time control : they don't support the use of a period time
-        // using absolute or simple (no period time) is likely to make bots timeout or play way too fast 
-        /* "fischer" , "simple", "byoyomi" , "canadian" , "absolute"*/
+        // ** first we eliminate "none" time control :
+        // infinite time is not supportable on OGS bots
+        /* "fischer" , "simple", "byoyomi" , "canadian" , "absolute", "none" */
         let TimeControlString = String(t.time_control);
-        if (TimeControlString === "absolute") {
-            conn_log("Minimum and Maximum main time is not supported in time control " + TimeControlString);
-            return { reject: true, msg: "Period time management is not supported in the time control " + TimeControlString + " , please choose a time control that supports the use of a period time, such as byoyomi,fischer,canadian." };
+        if (TimeControlString === "none") {
+            conn_log("The " + TimeControlString + " time control is impossible to manage for bots on OGS, not allowed : ");
```

Comment:
We do not get to decide if we allow "none" on ogs. This is up to the bot admin. Please leave this as a config option. (i can accept if the default does not allow none).

Reply:
ok, will just comment out this code then (maybe we want to change this policy in the future)